src/views/spotlights/housing/dashboard.vue

```diff
@@ -319,30 +319,10 @@ const updateCaseControls = (newControls) => {
   }
 };
 
-const dropDowns = {
-  age: {
-    icon: "fas fa-poll",
-    label: "What measure do you want to look at?",
-    values: [
-      { name: "Age Adjusted Rates", value: "adjusted" },
-      { name: "Rate by Age", value: "specific" },
-    ],
-  },
-};
-
 const hospFocus = { name: "Hospitalizations", value: "hospitalizations" };
 
 const controls = ref({
-  age: dropDowns.age.values[0],
-});
-
-useQueryParam({
```

Comment:
Hmm, we do want queryparams for both dropdowns

Reply:
Oh! If that's the case, then I think we should change the key, because I'm pretty sure it's only giving one option param as is 😅

Reply:
Right now, we have "age" and "cases" as keywords. "Age" is... not ideal there, though. This whole page certainly needs some cleanup! 
<img width="461" alt="image" src="https://github.com/pph-collective/signal-app/assets/45404173/78b1baa1-43ef-4f0a-8fb0-ea58c436befc">

Reply:
fixed 🚀